src/applications/survivors-benefits/config/chapters/04-household-information/dependentsPages.js

```diff
@@ -0,0 +1,457 @@
+import React from 'react';
+import { arrayBuilderPages } from '~/platform/forms-system/src/js/patterns/array-builder';
+import {
+  arrayBuilderItemFirstPageTitleUI,
+  arrayBuilderItemSubsequentPageTitleUI,
+  arrayBuilderYesNoUI,
+  arrayBuilderYesNoSchema,
+  fullNameUI,
+  fullNameSchema,
+  textUI,
+  textSchema,
+  yesNoUI,
+  yesNoSchema,
+  radioUI,
+  radioSchema,
+  ssnUI,
+  ssnSchema,
+  checkboxUI,
+  checkboxSchema,
+  currentOrPastDateUI,
+  currentOrPastDateSchema,
+  selectUI,
+  addressUI,
+  addressSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import {
+  STATE_NAMES,
+  STATE_VALUES,
+  COUNTRY_NAMES,
+  COUNTRY_VALUES,
+} from '../../../utils/labels';
+import {
+  DependentChildDescription,
+  seriouslyDisabledDescription,
+} from '../../../utils/helpers';
+import { VaForm214138Alert } from '../../../components/FormAlerts';
+
+/**
+ * Dependent children (array builder)
+ */
+
+const options = {
+  arrayPath: 'dependents',
+  nounSingular: 'dependent child',
+  nounPlural: 'dependent children',
+  required: false,
+  maxItems: 3,
+  isItemIncomplete: item => !item?.dependentFullName || !item?.dateOfBirth,
+  text: {
+    getItemName: item => {
+      if (
+        item &&
+        item.dependentFullName?.first &&
+        item.dependentFullName?.last
+      ) {
+        return `${item.dependentFullName.first} ${item.dependentFullName.last}`;
+      }
+      return 'Dependent';
+    },
+    cardDescription: () => '',
+  },
+};
+
+const introPage = {
+  uiSchema: {
+    ...arrayBuilderItemFirstPageTitleUI({
+      title: 'Dependents',
+      nounSingular: options.nounSingular,
+      nounPlural: options.nounPlural,
+    }),
+    'ui:description': () => (
+      <div>
+        <p className="vads-u-margin-top--0">
+          Next we’ll ask you about your dependent children. You may add up to 3
+          dependents.
+        </p>
+        {DependentChildDescription}
+      </div>
+    ),
+  },
+  schema: {
+    type: 'object',
+    properties: {},
+  },
+};
+
+const summaryPage = {
+  uiSchema: {
+    'ui:description': () => null,
+    'view:isAddingDependent': arrayBuilderYesNoUI(
+      options,
+      { hint: '' },
+      { hint: '' },
+      {
+        title: 'Do you have a dependent child to add?',
+      },
+    ),
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      'view:isAddingDependent': arrayBuilderYesNoSchema,
+    },
+    required: ['view:isAddingDependent'],
+  },
+};
+
+const namePage = {
+  uiSchema: {
+    ...arrayBuilderItemSubsequentPageTitleUI(
+      "Dependent's name and information",
+    ),
+    dependentFullName: fullNameUI(),
+    dependentSocialSecurityNumber: ssnUI(),
```

Comment:
This I think needs to be a conditional require, if I click the checkbox I can't continue without adding a SSN